### PR TITLE
Fix some typos through 'codespell'

### PR DESCRIPTION
**Pull request by kanurag94**
This is same as #110

### CODE REVIEW

File: catalogs/cs.cat [hunk 1]
```diff
@@ -669,7 +669,7 @@ decomp "binarry decomposition
 
 Pokud je imaginární část
 menší než nula, odečte se
-počet iterací od maximálního
+počet iterací of maximálního
```
Comment (arm-in) [inline catalogs/cs.cat]: My Czech is not very good, but I think "od" is correct

File: catalogs/de.cat [hunk 2]
```diff
@@ -21,7 +21,7 @@
 #Datei: dimension.xaf
 
 fmath "Die Mathematik hinter Fraktalen"
-fmath1 "Fraktale Geometrie ist ein sehr junges
+fmath1 "Fraktale Geometry ist ein sehr junges
```
Comment (arm-in) [inline catalogs/de.cat]: German "Geometrie" is correct.
Did you run "codespell" over foreign languages other than English? This won't work, most probably.

File: src/engine/formulas.cpp [hunk 3]
```diff
@@ -2,7 +2,7 @@
  *     XaoS, a fast portable realtime fractal zoomer
  *                  Copyright (C) 1996,1997 by
  *
- *      Jan Hubicka          (hubicka@paru.cas.cz)
+ *      Jan Hubicka          (hubicka@paru.case.cz)
```
Comment (arm-in) [inline src/engine/formulas.cpp]: You cannot "correct" an email address

File: src/engine/star.cpp [hunk 4]
```diff
@@ -23,16 +23,16 @@ static inline unsigned int myrandom(void)
 #define IMAGETYPE SMALLITER
 #include "c256.h"
 #define do_starfield do_starfield8
-#include "stard.h"
+#include "start.h"
```
Comment (arm-in) [inline src/engine/star.cpp]: You cannot "correct" the name of include files.

File: src/ui-hlp/menu.cpp [hunk 5]
```diff
@@ -375,7 +375,7 @@ static void uih_render(struct uih_context *c, dialogparam *d)
 static menudialog *uih_getcolordialog(struct uih_context *c)
 {
     if (c != NULL) {
-        uih_colordialog[0].defint = c->color;
+        uih_colordialog[0].definite = c->color;
```
Comment (arm-in) [inline src/ui-hlp/menu.cpp]: Correcting variable names is difficult, when not knowing where they emerge from.
I guess "defint" = "default value, integer"

**Comment by jblang** [comment]
I am sure there are some valid corrections here, but many of these are invalid. I won't be able to use this pull request as is.  It's "correcting" non-english words to english, changing an email address, changing the variable defint (default integer) to definite.

**Comment by jblang** [comment]
To be honest, I have my doubts about using an automated tool to do this. There are too many cases where it could mess things up. For example, "allways" is misspelled in a keyword for the XaoS file format, and if we correct it, it will break existing files saved from older versions. Another example, "similiar" is the key into a catalog file for a tutorial. Fixing it one place and not the other would break things.  I think it's better for humans to correct typos as they are found in the code and not take the chances with an automated tool which could break things.

**Comment by kanurag94** [comment]
That's true. I'll update the PR ans let you know

**Comment by jblang** [comment]
Thanks for the PR.

**Pull request merged**